dbt/include/bigquery/macros/adapters.sql

```diff
@@ -57,14 +66,18 @@
         {{ get_assert_columns_equivalent(compiled_code) }}
         {{ get_table_columns_and_constraints() }}
         {%- set compiled_code = get_select_subquery(compiled_code) %}
+      {% else %}
```

Comment:
It also looks weird to me... I wonder if was related to a previous merge because I don't recall why it would be there either 🤔

Reply:
Well I've spent absolutely way too much time trying to figure out what's going on here and why this control flow is necessary because honestly at first it looked like a bug to me.
> _PARTITIONDATE. Partition by ingestion time with daily partitions. This syntax cannot be used with the AS query_statement clause.
https://cloud.google.com/bigquery/docs/reference/standard-sql/data-definition-language#partition_expression

It seems that the syntax won't allow us to do model contracts and partition by at the same time